### DISCUSSION TITLE
Hello I have suggestion

### DISCUSSION THREAD

**Discussion by Touch1Me** [shaked6540/YoutubePlaylistDownloader#64] [Ideas]
maby u can add in the program to let us to login to youtube be cause my all playlist is in like vedeos so when i want to install doesn't let me be cause is private but youtube doesn't let me make public youtube remove that thing

**Comment by shaked6540** [comment]
I looked into it a while back and it looks like a hard thing to implement, both security wise and programming wise. 

The best thing I could implement would be a "1 time log in" where you just copy paste your cookies from the browser into my program and it will send it along with the requests, however, its not a good solution in my opinion because you will have to "log in" every time you want to use it.

You will need to extract your cookies using a browser extensions like https://github.com/lennonhill/cookies-txt for firefox or https://chrome.google.com/webstore/detail/get-cookiestxt/bgaddhkoddajcdgocldbbfleckgcbcid/ for chrome

I will think about it a bit more and will come with a final decision on the matter in the next 2 months, after my exams are over.

**Reply by Touch1Me** [reply]
I have to 2 suggestion 
1.make 1 more program where you need login to downlaud like vedeos but more mantaining needed
2.make program go through user browser where user is logined but i think program be slower 
 
I am not programmer so I guess how your program work and I don't know is hard to implement this things 
and sry if I sounded disrespetfull
